core/event.c

```diff
@@ -1415,6 +1429,12 @@ struct uwsgi_timer *event_queue_ack_timer(int id) {
 }
 #endif
 
+#if !HAS_ADD_FD_ACCEPT
```

Comment:
Which do you prefer: `#ifndef HAS_ADD_FD_ACCEPT ` or `#if !defined(HAS_ADD_FD_ACCEPT)` ?

FWIW, Microsoft recommends `defined`. They say `ifdef` and `ifndef` directives are provided only for compatibility with previous versions.
https://msdn.microsoft.com/ja-jp/library/2a1b21sf.aspx